samples/web/content/apprtc/room.py

```diff
@@ -65,6 +65,27 @@ def has_room(host, room_id):
   room = memcache_client.get(key)
   return room is not None
 
+def remove_room(host, room_id, client_id):
+  memcache_client = memcache.Client()
```

Comment:
I hadn't thought about it in terms of the actual caller/callee, just the race condition in the code.

I think A and B would both be callees. A is from accept, B is from decline.

I'll think this through a little more. It might make sense to move the Room creation to a more explicit call instead of being a side effect of the add_client update process.